### PR TITLE
docs(cpt): requirements section revamp

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Requirements sections updated for Cockpit pages.

### CODE REVIEW

**Review by nerda-codes — APPROVED** [review]

File: observability/cockpit/api-cli/configuring-grafana-agent.mdx [hunk 1]
```diff
@@ -15,13 +15,12 @@ categories:
...
 
-<Macro id="iam-requirements" />
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-  - You have installed [docker](https://www.docker.com/) and [docker compose](https://docs.docker.com/compose/install/)
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```
Comment (jcirinosclwy) [inline observability/cockpit/api-cli/configuring-grafana-agent.mdx:20]:
```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: observability/cockpit/how-to/access-grafana-and-managed-dashboards.mdx [hunk 2]
```diff
@@ -15,13 +15,12 @@ dates:
...
 
-<Macro id="iam-requirements" />
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-  - You have [retrieved your Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/)
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/access-grafana-and-managed-dashboards.mdx:20]:
```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 3]
```diff
@@ -15,15 +15,16 @@ dates:
 
 This page shows you how to activate and monitor your Scaleway products from your Cockpit using the [Scaleway console](https://console.scaleway.com/).
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/activate-cockpit.mdx:20]:
```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: observability/cockpit/troubleshooting/pusher-does-not-support-custom-http-headers.mdx [hunk 4]
```diff
@@ -13,7 +13,10 @@ categories:
   - observability
 ---
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```
Comment (jcirinosclwy) [inline observability/cockpit/troubleshooting/pusher-does-not-support-custom-http-headers.mdx:18]:
```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx [hunk 5]
```diff
@@ -16,12 +16,11 @@ dates:
...
 You can push logs with any Loki compatible agent such as the [Promtail](https://grafana.com/docs/loki/latest/clients/promtail/), [Fluentd](https://docs.fluentd.org/), [Fluent Bit](https://docs.fluentbit.io/manual/) or [Logstash](https://www.elastic.co/guide/en/logstash/current/introduction.html) agents.
 
-<Macro id="iam-requirements" />
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx:21]:
```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: observability/cockpit/how-to/activate-push-traces.mdx [hunk 6]
```diff
@@ -14,12 +14,11 @@ dates:
...
 This page shows you how to activate [traces](/observability/cockpit/concepts/#traces) from the Scaleway console, retrieve the traces push paths, configure the OpenTelemetry HTTP agent, and push traces.
 
-<Macro id="iam-requirements" />
+## Requirements
 
-<Message type="requirement">
- - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
- - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit)
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/activate-push-traces.mdx:19]:
```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 7]
```diff
@@ -15,15 +15,16 @@ dates:
 
 This page shows you how to activate and monitor your Scaleway products from your Cockpit using the [Scaleway console](https://console.scaleway.com/).
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/activate-cockpit.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: observability/cockpit/how-to/activate-managed-alerts.mdx [hunk 8]
```diff
@@ -14,10 +14,11 @@ dates:
 
 This page shows you how to activate [managed alerts](/observability/cockpit/concepts/#managed-alerts) using the [Scaleway console](https://console.scaleway.com/).
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-</Message>
+## Requirements
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/activate-managed-alerts.mdx:17]:
```suggestion
<Macro id="requirements" />
```

File: observability/cockpit/how-to/activate-managed-alerts.mdx [hunk 9]
```diff
@@ -14,10 +14,11 @@ dates:
...
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/activate-managed-alerts.mdx:20]:
```suggestion
- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```

File: observability/cockpit/how-to/activate-push-traces.mdx [hunk 10]
```diff
@@ -14,12 +14,11 @@ dates:
 
 This page shows you how to activate [traces](/observability/cockpit/concepts/#traces) from the Scaleway console, retrieve the traces push paths, configure the OpenTelemetry HTTP agent, and push traces.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/activate-push-traces.mdx:17]:
```suggestion
<Macro id="requirements" />
```

File: observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx [hunk 11]
```diff
@@ -14,14 +14,15 @@ dates:
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-  - You have created Scaleway resources to monitor
-  - You have [created your Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/) and selected the **Editor** role
-  - You have [activated managed alerts and created at least one contact point](/observability/cockpit/how-to/activate-managed-alerts/)
-  - You are using the **Scaleway Alerting** alert manager in Grafana
-</Message>
+## Requirements
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx:17]:
```suggestion
<Macro id="requirements" />
```

File: observability/cockpit/how-to/deactivate-cockpit.mdx [hunk 12]
```diff
@@ -15,18 +15,15 @@ dates:
 
 This page shows you how to deactivate your Cockpit.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/deactivate-cockpit.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 13]
```diff
@@ -15,13 +15,10 @@ dates:
 
 This page shows you how to retrieve your Grafana credentials to access your Grafana [dashboards](/observability/cockpit/concepts/#managed-dashboards/) using the [Scaleway console](https://console.scaleway.com/).
 
-<Macro id="iam-requirements" />
- 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-</Message>
+## Requirements
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: observability/cockpit/quickstart.mdx [hunk 14]
```diff
@@ -23,11 +23,10 @@ Read [our documentation](/observability/cockpit/reference-content/cockpit-limita
 
 In this quickstart, we show you how to activate your Cockpit, how to retrieve your Grafana credentials and how to create a push token to send metrics and logs to your Cockpit.
 
-<Macro id="iam-requirements" />
- 
-<Message type="requirement">
-  You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-</Message>
+## Requirements
```
Comment (jcirinosclwy) [inline observability/cockpit/quickstart.mdx:26]:
```suggestion
<Macro id="requirements" />
```

File: observability/cockpit/troubleshooting/pusher-does-not-support-custom-http-headers.mdx [hunk 15]
```diff
@@ -13,7 +13,10 @@ categories:
   - observability
 ---
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline observability/cockpit/troubleshooting/pusher-does-not-support-custom-http-headers.mdx:16]:
```suggestion
<Macro id="requirements" />
```

File: observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx [hunk 16]
```diff
@@ -16,12 +16,11 @@ dates:
...
-<Macro id="iam-requirements" />
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-</Message>
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx:23]:
```suggestion
- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 17]
```diff
@@ -15,13 +15,10 @@ dates:
...
 
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:21]:
```suggestion
- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```

File: observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx [hunk 18]
```diff
@@ -14,14 +14,15 @@ dates:
...
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx:20]:
```suggestion
- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```

File: observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx [hunk 19]
```diff
@@ -14,14 +14,15 @@ dates:
...
+- Have created Scaleway resources to monitor
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx:21]:
```suggestion
- Scaleway resources to monitor
```

File: observability/cockpit/how-to/activate-push-traces.mdx [hunk 20]
```diff
@@ -14,12 +14,11 @@ dates:
...
 
-<Message type="requirement">
- - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
- - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit)
-</Message>
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```
Comment (jcirinosclwy) [inline observability/cockpit/how-to/activate-push-traces.mdx:21]:
```suggestion
- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```

File: observability/cockpit/how-to/create-token.mdx [hunk 21]
```diff
@@ -15,13 +15,12 @@ dates:
...
- - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
- - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
- - You have [activated traces](/observability/cockpit/how-to/activate-push-traces) if you want to create a token to push [traces](/observability/cockpit/concepts/#traces)
-</Message>
+<Macro id="requirements" />
+
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+- Have [activated traces](/observability/cockpit/how-to/activate-push-traces) if you want to create a token to push [traces](/observability/cockpit/concepts/#traces)
```
Comment (RoRoJ) [inline observability/cockpit/how-to/create-token.mdx:23]:
```suggestion
- [Activated traces](/observability/cockpit/how-to/activate-push-traces) if you want to create a token to push [traces](/observability/cockpit/concepts/#traces)
```

File: observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx [hunk 22]
```diff
@@ -14,14 +14,15 @@ dates:
...
-  - You have [activated managed alerts and created at least one contact point](/observability/cockpit/how-to/activate-managed-alerts/)
-  - You are using the **Scaleway Alerting** alert manager in Grafana
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+- Scaleway resources you can monitor
+- Grafana [credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/) with the **Editor** role
+- Have [activated managed alerts and created at least one contact point](/observability/cockpit/how-to/activate-managed-alerts/)
```
Comment (RoRoJ) [inline observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx:23]:
```suggestion
- [Activated managed alerts and created at least one contact point](/observability/cockpit/how-to/activate-managed-alerts/)
```

File: observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx [hunk 23]
```diff
@@ -14,14 +14,15 @@ dates:
...
+- Use the **Scaleway Alerting** alert manager in Grafana
```
Comment (RoRoJ) [inline observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx:24]: Don't think this works with the "To complete the actions presented on this page, you must have..." intro,  but not sure what it should be?

File: observability/cockpit/api-cli/configuring-grafana-agent.mdx [hunk 24]
```diff
@@ -15,13 +15,12 @@ categories:
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-  - You have installed [docker](https://www.docker.com/) and [docker compose](https://docs.docker.com/compose/install/)
-</Message>
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+- [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/)
```
Comment (RoRoJ) [inline observability/cockpit/api-cli/configuring-grafana-agent.mdx:23]:
```suggestion
- Installed [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/)
```

File: observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx (hunk 23 above)
Comment (nerda-codes) [reply observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx:24]: "Selected the **Scaleway Alerting** alert manager in Grafana"

**Pull request closed without merging**